File: openai-api/in-js/chatgpt-api.js
import readline from "readline"
import { OpenAIApi, Configuration } from "openai"
import { config } from "dotenv"
config({path: '../.env'})

console.log("Welcome to ChatGPT-586")

const userInterface = readline.createInterface({
    input: process.stdin,
    output: process.stdout
})

const configuration = new Configuration({
    apiKey: process.env.OPENAI_API_KEY
})
const openai = new OpenAIApi(configuration)

userInterface.prompt()
userInterface.on("line", async input => {
    const res = openai.createChatCompletion({
        model: "gpt-3.5-turbo",
        messages: [{
            role: 'user',
            content: input
        }] 
    })
    console.log((await res).data.choices[0].message.content)
    userInterface.prompt()
})

File: openai-api/in-python/chatgpt-api.py
import os
import openai 
from dotenv import load_dotenv

load_dotenv('../.env')

openai.api_key = os.getenv("OPENAI_API_KEY")


print("Welcome to ChatGPT-586")
message = input("> ")

while message != "quit()" :
    chat_completion = openai.ChatCompletion.create(
        model="gpt-3.5-turbo", 
        messages=[{
            "role": "user", 
            "content": message
        }]
    )
    print(chat_completion.choices[0].message.content)
    message = input("> ")
